frontend/src/features/public-form/components/FormPaymentPage/stripe/components/PaymentItemDetailsBlock.tsx

```diff
@@ -26,9 +26,9 @@ export const PaymentItemDetailsBlock = ({
       <Text textStyle="body-1" mb="0.75rem">
         {paymentItemName}
       </Text>
-      <Box as="h2" textStyle="h2">{`${centsToDollars(
+      <Box as="h2" textStyle="h2">{`S$ ${centsToDollars(
```

Comment:
<img width="1033" alt="image" src="https://github.com/opengovsg/FormSG/assets/59867455/8df08dc8-0f8d-44aa-a336-6b29216bc8ec">

@staceytan1998 can check which version (the above) or @mayying version with the space distance. Not sure which is more palatable! Cause on the figma it seems to have a small space but not large enough for a full h2 space😅

Reply:
I think stacey meant for it to be spaceless, i have amended it, thanks!